data/projects.json

```diff
@@ -198,5 +198,15 @@
         "level": "intermediate",
         "site": "https://poketalog.herokuapp.com/",
         "tags": "pokeapi, pokemon, vue, catalog"
+    },
+    {
+        "name": "kunci web scrapper",
```

Comment:
Hi @shonenxnaifu,
The `name` property is used by repository name on Github, so please check if the repo is public and the name is correct.

Reply:
helo @ming-tsai i've been changed the name and display name, the repository is already public, thank you